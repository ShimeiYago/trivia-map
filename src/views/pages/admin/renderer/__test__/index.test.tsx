import { shallow, ShallowWrapper } from 'enzyme';
import { Renderer, Props } from '..';

let wrapper: ShallowWrapper<Props, unknown, Renderer>;

const basicProps: Props = {
  user: {
    userId: 1,
    nickname: 'Axel',
    email: 'xxx@example.com',
  },
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<Renderer {...basicProps} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
