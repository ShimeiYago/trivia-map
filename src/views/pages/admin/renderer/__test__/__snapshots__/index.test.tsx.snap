// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<ForwardRef(Box)
  sx={
    Object {
      "backgroundColor": "#f4f5f7",
    }
  }
>
  <GlobalMenu
    permanentLeftNavi={true}
    topBarPosition="static"
  >
    <ForwardRef(Box)
      sx={
        Object {
          "marginTop": 5,
          "marginX": 10,
        }
      }
    >
      <ForwardRef(Grid)
        container={true}
        spacing={4}
      >
        <ForwardRef(Grid)
          item={true}
          xs={12}
        >
          <ArticlePaper
            variant="main"
          >
            <ForwardRef(Stack)
              spacing={2}
            >
              <ForwardRef(Box)>
                ようこそ 
                Axel
                さん
              </ForwardRef(Box)>
            </ForwardRef(Stack)>
          </ArticlePaper>
        </ForwardRef(Grid)>
      </ForwardRef(Grid)>
    </ForwardRef(Box)>
  </GlobalMenu>
</ForwardRef(Box)>
`;

exports[`Shallow Snapshot Tests loading 1`] = `
<ForwardRef(Box)
  sx={
    Object {
      "mt": 3,
    }
  }
>
  <CenterSpinner />
</ForwardRef(Box)>
`;

exports[`Shallow Snapshot Tests mobile 1`] = `
<ForwardRef(Box)
  sx={
    Object {
      "backgroundColor": "#f4f5f7",
    }
  }
>
  <GlobalMenu
    permanentLeftNavi={false}
    topBarPosition="static"
  >
    <ForwardRef(Box)
      sx={
        Object {
          "marginTop": 1,
          "marginX": 0,
        }
      }
    >
      <ForwardRef(Grid)
        container={true}
        spacing={2}
      >
        <ForwardRef(Grid)
          item={true}
          xs={12}
        >
          <ArticlePaper
            variant="main"
          >
            <ForwardRef(Stack)
              spacing={2}
            >
              <ForwardRef(Box)>
                ようこそ 
                Axel
                さん
              </ForwardRef(Box)>
            </ForwardRef(Stack)>
          </ArticlePaper>
        </ForwardRef(Grid)>
      </ForwardRef(Grid)>
    </ForwardRef(Box)>
  </GlobalMenu>
</ForwardRef(Box)>
`;

exports[`Shallow Snapshot Tests redirect 1`] = `
<Navigate
  to="/login"
/>
`;
